pipeline/run_beam_tables.py

```diff
@@ -158,6 +167,11 @@ def parse_args() -> argparse.Namespace:
       default=None,
       metavar='yyyy-mm-dd',
       help='Last date to read data from')
+  parser.add_argument(
+      '--export_gcs',
```

Comment:
We're going to want a way to run these both simultaneously (so we can produce both without having to redo the internal pipeline work.

That's complex to deal with now when the output tables and files are out of sync. But in the future we'll want them to be in sync at all times. Maybe just add a todo for now while we're developing.

Reply:
Added a TODO